src/Type/Constant/ConstantIntegerType.php

```diff
@@ -74,7 +76,10 @@ public function toFloat(): Type
 
 	public function toString(): Type
 	{
-		return new ConstantStringType((string) $this->value);
+		return new IntersectionType([
+			new ConstantStringType((string) $this->value),
+			new AccessoryNumericStringType(),
```

Comment:
This isn't necessary, `ConstantStringType` will already say that it's numeric based on the value.